e2e/tests/custom-projects/create-custom-project.spec.ts

```diff
@@ -1,52 +1,276 @@
-import { Page, test, expect } from "@playwright/test";
+import { Page, test, expect, Locator } from "@playwright/test";
+import { ACTIVITY } from "@shared/entities/activity.enum";
 import { E2eTestManager } from "@shared/lib/e2e-test-manager";
+import { EXTENDED_TIMEOUT, ROUTES } from "e2e/constants";
 
 let testManager: E2eTestManager;
 let page: Page;
 
 test.describe.configure({ mode: "serial" });
 
-test.describe("Custom Projects", () => {
-  const projectName = "test project";
-  const insertProjectName = async () => {
-    await page.getByRole("textbox", { name: "Insert project name" }).click();
-    await page
-      .getByRole("textbox", { name: "Insert project name" })
-      .fill(projectName);
-  };
-  const submitCustomProjectAndCheckPreview = async () => {
-    await page.getByRole("button", { name: "Continue" }).click();
-    await page.waitForURL("/projects/preview");
-    await expect(
-      page.getByRole("heading", { name: projectName }),
-    ).toBeVisible();
-  };
-
+test.describe("Custom Projects - Create", () => {
   test.beforeAll(async ({ browser }, testInfo) => {
-    testInfo.setTimeout(testInfo.timeout + 60_000);
+    testInfo.setTimeout(testInfo.timeout + EXTENDED_TIMEOUT);
     page = await browser.newPage();
     testManager = await E2eTestManager.load(page);
     await testManager.ingestBaseData();
```

Comment:
Reminder to myself to bring this up later, as it is related to another topic I want to discuss with you all